src/Exp.js

```diff
@@ -108,7 +155,7 @@ class Exp {
                if (method === null || !(method in that.methods)) return;
 
                el.addEventListener(event, function() {
-                   that.methods[method].apply(that.model);
+                   that.methods[method].apply(that.model, [el]);
```

Comment:
@hackal I have tried passing the `event` as an argument, but it only passed string, e.g. `"click"`. We can pass event as well as `el`, but I think that passing `el` is definitely worth it, at least in `exp-for` cases.